backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/appsearchmodal/BpkAppSearchModal.kt

```diff
@@ -95,6 +97,7 @@ fun BpkAppSearchModal(
             inputHint = inputHint,
             results = results,
             onInputChanged = onInputChanged,
+            clearAction = clearAction,
```

Comment:
we should also make sure that we're only showing the clear action if the text is not empty here